### PR TITLE
[Telemetry] Bump redirect for VSTelemetry

### PR DESCRIPTION
**Pull request by sgmunn**

### CODE REVIEW

File: main/src/core/MonoDevelop.Startup/app.config [hunk 1]
```diff
@@ -26,7 +26,7 @@
       <dependentAssembly><assemblyIdentity name="Microsoft.VisualStudio.Text.Data" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" /> <bindingRedirect oldVersion="0.0.0.0-16.0.0.0" newVersion="16.0.0.0" /></dependentAssembly>
       <dependentAssembly><assemblyIdentity name="Microsoft.VisualStudio.Text.Logic" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" /> <bindingRedirect oldVersion="0.0.0.0-16.0.0.0" newVersion="16.0.0.0" /></dependentAssembly>
       <dependentAssembly><assemblyIdentity name="Microsoft.VisualStudio.Text.UI" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" /> <bindingRedirect oldVersion="0.0.0.0-16.0.0.0" newVersion="16.0.0.0" /></dependentAssembly>
-      <dependentAssembly><assemblyIdentity name="Microsoft.VisualStudio.Telemetry" culture="neutral" publicKeyToken="b03f5f7f11d50a3a" /><bindingRedirect oldVersion="0.0.0.0-16.0.0.0" newVersion="16.0.0.0" /></dependentAssembly>
+      <dependentAssembly><assemblyIdentity name="Microsoft.VisualStudio.Telemetry" culture="neutral" publicKeyToken="b03f5f7f11d50a3a" /><bindingRedirect oldVersion="0.0.0.0-16.0.0.0" newVersion="16.3.2.0" /></dependentAssembly>
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Startup/app.config]:
```suggestion
      <dependentAssembly><assemblyIdentity name="Microsoft.VisualStudio.Telemetry" culture="neutral" publicKeyToken="b03f5f7f11d50a3a" /><bindingRedirect oldVersion="0.0.0.0-16.3.2.0" newVersion="16.3.2.0" /></dependentAssembly>
```

Comment (mrward) [reply main/src/core/MonoDevelop.Startup/app.config]: This is not needed. The assembly version has not changed.

**Comment by mrward** [comment]
```
[assembly: AssemblyFileVersion ("16.3.2.14029")]
[assembly: AssemblyInformationalVersion ("16.3.2+gcd36577e57")]
[assembly: TargetFramework (".NETFramework,Version=v4.5", FrameworkDisplayName = ".NET Framework 4.5")]
[assembly: SecurityPermission (SecurityAction.RequestMinimum, SkipVerification = true)]
[assembly: AssemblyVersion ("16.0.0.0")]
```

**Comment by Therzok** [comment]
Closing as per https://github.com/mono/monodevelop/pull/8054#issuecomment-506483766

**Pull request closed without merging**